openqaoa/optimizers/qaoa_optimizer.py

```diff
@@ -55,7 +56,8 @@ def get_optimizer(vqa_object: VQABaseBackend,
     """
     SUPPORTED_OPTIMIZERS = {
         'scipy': ScipyOptimizer,
-        'custom_scipy_gradient': CustomScipyGradientOptimizer
+        'custom_scipy_gradient': CustomScipyGradientOptimizer,
+        'custom_scipy_pennylane': CustomScipyPennyLaneOptimizer
```

Comment:
@raulconchello, let's also change this. Let's just simply call them Pennylane optimizers. We can remove the custom scipy naming